rhel.Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
+FROM rhel8/go-toolset:1.11.13-18 as builder
+ENV PATH=/opt/rh/go-toolset-1.11/root/usr/bin:$PATH \
+    GOPATH=/go/
+USER root
+WORKDIR /go/src/github.com/eclipse/che-machine-exec/
+COPY . .
+RUN adduser unprivilegeduser && \
+    CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -a -installsuffix cgo -o che-machine-exec .
+
+FROM scratch
+ENV SUMMARY="Red Hat CodeReady Workspaces machine exec container" \
```

Comment:
why this has been merged and approved
It contains Code Ready stuff that is not belonging into Eclipse Che repository

Please revert, thanks

Reply:
reverted in https://github.com/eclipse/che-machine-exec/commit/19c1fed64fcecf56b10a8500f02869b889231974